applications/GeoMechanicsApplication/tests/test_quad_4n.gid/ProjectParameters.json

```diff
@@ -80,7 +80,7 @@
                         "gidpost_flags":       {
                             "WriteDeformedMeshFlag": "WriteUndeformed",
                             "WriteConditionsFlag":   "WriteElementsOnly",
-                            "GiDPostMode":           "GiD_PostBinary",
+                            "GiDPostMode":           "GiD_PostAscii",
```

Comment:
Is there a reason to change binary to ascii? As long as we don't need to look at the values inside, binary is preffered, as it leads to compacter file.

Reply:
Helper class `GidOutputFileReader` can only handle GiD output files in text format. Since we now use that helper class in our tests (for checking the displacement fields), we must use textual output files rather than binary ones.